svg.js.d.ts

```diff
@@ -1382,6 +1388,13 @@ declare module "@svgdotjs/svg.js" {
         target(): string;
     }
 
+
+    // ForeignObject.js
+    class ForeignObject extends Element {
+        constructor(node: HTMLElement, attrs?: any)

```

Comment:
```suggestion
        constructor(node?: SVGForeignObjectElement, attrs?: object)
        constructor(attrs?: object)
```

This is wrong in other places, too. But thats how it should be